InteractionBase/src/vis/VViewSwitcherEntry.h

```diff
@@ -0,0 +1,59 @@
+/***********************************************************************************************************************
+ **
+ ** Copyright (c) 2015 ETH Zurich
+ ** All rights reserved.
+ **
+ ** Redistribution and use in source and binary forms, with or without modification, are permitted provided that the
+ ** following conditions are met:
+ **
+ **    * Redistributions of source code must retain the above copyright notice, this list of conditions and the
+ **      following disclaimer.
+ **    * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the
+ **      following disclaimer in the documentation and/or other materials provided with the distribution.
+ **    * Neither the name of the ETH Zurich nor the names of its contributors may be used to endorse or promote products
+ **      derived from this software without specific prior written permission.
+ **
+ **
+ ** THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+ ** INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ ** DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ ** SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ ** SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+ ** WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ ** OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ **
+ **********************************************************************************************************************/
+#pragma once
+
+#include "interactionbase_api.h"
+#include "VisualizationBase/src/declarative/DeclarativeItem.h"
+#include "VisualizationBase/src/items/Text.h"
+
+namespace Interaction {
+
+class TextAndDescription;
+
+class INTERACTIONBASE_API VViewSwitcherEntry :
+		public Super<Visualization::DeclarativeItem<VViewSwitcherEntry>>
+{
+	ITEM_COMMON_CUSTOM_STYLENAME(VViewSwitcherEntry, Visualization::DeclarativeItemBaseStyle)
+
+	public:
+		VViewSwitcherEntry(Visualization::Item* parent, QString viewName,
+						   const StyleType* style = itemStyles().get());
+
+		static void initializeForms();
+
+		Visualization::Text* nameField() const;
+
+		virtual void determineChildren() override;
+
+	private:
+		Visualization::Text* nameField_{};
```

Comment:
Last time I suggested you use VText instead of Text. Could you please try that? If you have tried it, was there a problem?